src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+import { sleep } from '../../services/utils'
+import { repeatArray } from '../../services/utils'
+import { User } from './../../pages/users/types'
+
+export const users = repeatArray<User>(50, [
+  {
+    id: 1,
+    fullname: 'Patrik Radkow',
+    email: 'magicpan@example.gg',
+    username: 'magicpan',
+    role: 'user',
+    projects: 3,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 2,
+    fullname: 'Martin Hoff',
+    email: 'niceadmin@mail.com',
+    username: 'admin',
+    role: 'admin',
+    projects: 5,
+    avatar: '😍',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 3,
+    fullname: 'Liz Macintosh',
+    email: 'ebrown@gmail.com',
+    username: 'ebrown',
+    role: 'user',
+    projects: 2,
+    avatar: 'https://randomuser.me/api/portraits/men/1.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 4,
+    fullname: 'M2',
+    email: 'mrm@gmail.com',
+    username: 'mrm',
+    role: 'owner',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 5,
+    fullname: 'Kevin Smith',
+    email: 'kevin@gmail.com',
+    username: 'kevin13',
+    role: 'user',
+    projects: 13,
+    avatar: 'https://randomuser.me/api/portraits/men/2.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 6,
+    fullname: 'Martin Hoff',
+    email: 'martin@gmail.com',
+    username: 'martin3',
+    role: 'user',
+    projects: 2,
+    avatar: 'https://randomuser.me/api/portraits/men/3.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 7,
+    fullname: 'John Doe',
+    email: 'john@mail.com',
+    username: 'john',
+    role: 'user',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: '',
+  },
+  {
+    id: 8,
+    fullname: 'Maksim Nedo',
+    email: 'maksim@epic.com',
+    username: 'maksim',
+    role: 'admin',
+    projects: 1,
```

Comment:
![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/76878ee4-37ad-410d-9508-6d2a97f6e691)

Let's use project names.
I'm fine without objects, but the way it is right now - I can't say the page is finished. So let's make it look good in this PR, then perhaps move to objects in #984 pr.